lib/grape_fast_jsonapi/parser.rb

```diff
@@ -0,0 +1,175 @@
+module GrapeSwagger
+  module FastJsonapi
+    class Parser
+      attr_reader :model
+      attr_reader :endpoint
+
+      def initialize(model, endpoint)
+        @model = model
+        @endpoint = endpoint
+      end
+
+      def call
```

Comment:
@EmCousin cool! I tried to take a crack at this yesterday but failed miserably hahah